### PR TITLE
Add word-break: auto-phrase as of Chrome 119

### PR DESCRIPTION
**Pull request by Psychpsyo**
#### Summary

Chrome 119 supports ``word-break: auto-phrase`` for Japanese text.

#### Test results and supporting details

Announced on the Chrome developer blog here: https://developer.chrome.com/blog/css-i18n-features

#### Related issues

None

### CODE REVIEW

File: css/properties/word-break.json [hunk 1]
```diff
@@ -115,6 +115,30 @@
...
             }
           }
+        },
+        "auto-phrase": {
+          "__compat": {
+            "description": "<code>auto-phrase</code>",
+            "support": {
+              "chrome": {
+                "version_added": "119",
+                "notes": "Only works if Japanese is declared with lang=\"ja\" and does not affect other languages."
```
Comment (queengooborg) [inline css/properties/word-break.json:125]:
```suggestion
                "notes": "This value is only applicable if <code>lang=\"ja\"</code> is specified. This value has no effect on other locales."
```

File: css/properties/word-break.json [hunk 2]
```diff
@@ -115,6 +115,30 @@
...
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": true
+              },
```
Comment (queengooborg) [inline css/properties/word-break.json:130]:
```suggestion
              "edge": "mirror",
```

File: css/properties/word-break.json [hunk 3]
```diff
@@ -115,6 +115,30 @@
...
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror"
```
Comment (queengooborg) [inline css/properties/word-break.json:134]: All other browsers must also be added, including Safari, Samsung Internet, etc.

**Review by queengooborg — CHANGES_REQUESTED** [review]
Hey there, thank you for your PR and welcome to BCD!  Got a couple of change requests for better flow of notes and to fix linter issues:

File: css/properties/word-break.json (hunk 3 above)
Comment (Psychpsyo) [reply css/properties/word-break.json]: I've added them now.

File: css/properties/word-break.json [hunk 4]
```diff
@@ -46,6 +46,44 @@
...
+              },
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": {
+                "version_added": false
+              },
```
Comment (queengooborg) [inline css/properties/word-break.json:77]:
```suggestion
              "samsunginternet_android": "mirror",
```

File: css/properties/word-break.json [hunk 5]
```diff
@@ -46,6 +46,44 @@
...
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": {
+                "version_added": false
+              },
```
Comment (queengooborg) [inline css/properties/word-break.json:68]:
```suggestion
              "oculus": "mirror",
```

**Review by queengooborg — CHANGES_REQUESTED** [review]

**Review by queengooborg — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**